### ISSUE TITLE
Broken Unicode in ThirdPartyNoticeText.txt

### ISSUE THREAD

**Issue by gforcada** [microsoft/TypeScript#52778]
# Bug Report

Seems that `ThirdPartyNoticeText.txt` has some bad unicode characters, at least python 3 fails to parse it 😅 

```shell
$ cd /tmp
$ git clone --depth 1 https://github.com/microsoft/typescript
$ python3.11
>>> with open('typescript/ThirdPartyNoticeText.txt') as file_handle:
...     for line in file_handle.readlines():
...         print()
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa0 in position 6038: invalid start byte
```

I could provide a PR to fix it, if such a PR is welcomed 😄

**Comment by RyanCavanaugh** [comment]
Looks like some smart quotes got mangle around "you" / "your"

**Comment by JoshuaKGoldberg** [comment]
This was fixed by #53184
